client/src/components/NavBar.js

```diff
@@ -20,17 +20,22 @@ export default function NavBar() {
 	};
 
 	const handleLogout = () => {
-		if(window.confirm('You sure you want to logout?')){
+		if (window.confirm("You sure you want to logout?")) {
 			localStorage.clear();
 			history.push("/login");
 		}
 	};
 
+	const goToAccount = () => {
+		history.push("/manage-account");
+	};
+
 	return <div className={"navbar__container"}>
 		<button class="navbar__container__button" onClick={goToHome}>Home</button>
 		<button class="navbar__container__button" onClick={goToProfile}>Profile</button>
 		<button class="navbar__container__button" onClick={goToWeather}>Weather</button>
 		<button class="navbar__container__button" onClick={handleLogout}>Logout</button>
+		<button class="navbar__container__button" onClick={goToAccount}>Account Management</button>
```

Comment:
I don't think we want account management in the nav bar. It should be a button that shows up when viewing your profile.

Reply:
Honestly, I think it's fine to keep this as is -- maybe we could rename to page to "Settings" for simplicity instead of "Account Management".

My vote is to keep it in the nav bar.